docs/user/03-30-management-of-service-instances-and-bindings.md

```diff
@@ -0,0 +1,189 @@
+
+3.  Verify the Secret is created with the name specified in the  **spec.secretName** field of the ServiceBinding CR. The Secret contains access credentials that the applications need to use the service:
+
+    ```bash
+    kubectl get secrets {BINDING_NAME} -n {NAMESPACE}
+    NAME              TYPE     DATA   AGE
+    {BINDING_NAME}    Opaque   5      32s
+    ```
+
+    To learn about different options for using the credentials from your application running in the Kubernetes cluster, see [Uses for Secrets](https://kubernetes.io/docs/concepts/configuration/secret/#uses-for-secrets).
+
+## Pass Parameters
+
+To set input parameters, go to the `spec` of the ServiceInstance or ServiceBinding resource, and use both or one of the following fields:
+* **parameters**: Specifies a set of properties sent to the service broker.
+  The specified data is passed to the service broker without any modifications - aside from converting it to JSON for transmission to the broker if the `spec` field is specified as YAML.
+  All valid YAML or JSON constructs are supported. 
+  > [!NOTE] 
+  > Only one parameter field per `spec` can be specified.
+* **parametersFrom**: Specifies which Secret, together with the key in it, to include in the set of parameters sent to the service broker.
+  The key contains a `string` that represents the JSON. The **parametersFrom** field is a list that supports multiple sources referenced per `spec`.
+
+If you specified multiple sources in the **parameters** and **parametersFrom** fields, the final payload results from merging all of them at the top level.
+If there are any duplicate properties defined at the top level, the specification is considered to be invalid. 
+The further processing of the ServiceInstance or ServiceBinding resource stops with the status `Error`.
+
+See the following example of the `spec` format in YAML:
+
+```yaml
+spec:
+  ...
+  parameters:
+    name: value
+  parametersFrom:
+    - secretKeyRef:
+        name: {SECRET_NAME}
+        key: secret-parameter
+```
+
+See the following example of the `spec` format in JSON:
+
+```json
+{
+  "spec": {
+    "parameters": {
+      "name": "value"
+    },
+    "parametersFrom": {
+      "secretKeyRef": {
+        "name": "{SECRET_NAME}",
+        "key": "secret-parameter"
+      }
+    }
+  } 
+}
+```
+
+See the exampple of a Secret with the key named **secret-parameter**:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {SECRET_NAME}
+type: Opaque
+stringData:
+  secret-parameter:
+    '{
+      "password": "password"
+    }'
+```
+
+See the example of the final JSON payload sent to the service broker:
+```json
+{
+  "name": "value",
+  "password": "password"
+}
+```
+
+To list multiple parameters in the Secret, separate key-value pairs with commas. See the following example:
+
+```yaml
+secret-parameter:
+  '{
+    "password": "password",
+    "key2": "value2",
+    "key3": "value3"
+  }'
+```
+
+## Delete Service Bindigs and Service Instances
+
```

Comment:
this section has 4 paragraphs - one of them is normal text, the other three are a warning (which makes sense to me) and a note. This feels off balance to me; notes/warnings should be the exception rather than the norm. Are you sure the note can't be a normal paragraph?